skin/src/vue/c-sidebar.vue

```diff
@@ -23,33 +23,122 @@
  *
 -->
 <script lang="ts" setup>
+import { Ref, onMounted, onUnmounted, ref, watch } from "vue";
 import CConfigMenu from "./c-config-menu.vue";
+import CNavigationDrawer from "./c-navigation-drawer.vue";
 import CSidebarPanel from "./c-sidebar-panel.vue";
 import CHelp from "./c-help.vue";
 import xlogo from "../images/xwiki-logo-color.svg";
 import xavatarImg from "../images/no-one.svg";
 import { CIcon } from "@xwiki/cristal-icons";
+import { useMouse } from "../composables/mouse";
+import { ViewportType, useViewportType } from "../composables/viewport";
 
 const logo = xlogo;
 const avImg = xavatarImg;
+const viewportSize: Ref<ViewportType> = useViewportType();
```

Comment:
I feel like `viewportType` is more accurate here (and also in `useViewportType` implementation)
```suggestion
const viewportType: Ref<ViewportType> = useViewportType();
```